src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -193,6 +194,7 @@ def _get_process_info(self, index):
         start_time = time.time()
         while True:
             try:
+                self._processes_manager.ensure_healthy()
```

Comment:
Put `ensure_healthy` outside try? So we don't need to specially handle the PromptflowException

Reply:
If we put ensure_healthy outside, it will return true and end after 1 minute.

Reply:
I mean something like this:
```
while True:
    self._process_manager.ensure_healthy()
    try:
        xxxx
    except
        xxxx
```

Reply:
fixed